content/logo/index.html.haml

```diff
@@ -0,0 +1,80 @@
+---
+layout: default
+title: Jenkins Logos
```

Comment:
Similarly, we might mention that only one is the official logo, as mentioned on the Press page:
https://jenkins.io/press/#artwork